README.md

```diff
@@ -97,14 +97,18 @@ Error in simulation (with mitigation): 0.000519
 
 ## Error mitigation techniques
 
-Mitiq currently implements [zero-noise extrapolation](https://mitiq.readthedocs.io/en/stable/guide/guide-zne.html) and 
+Mitiq currently implements [zero-noise extrapolation](https://mitiq.readthedocs.io/en/stable/guide/guide-zne.html) and
 is designed to support [additional techniques](https://github.com/unitaryfund/mitiq/wiki).
 
 ## Documentation
 
-Mitiq's documentation is hosted at [mitiq.readthedocs.io](https://mitiq.readthedocs.io). A PDF version of the latest 
+Mitiq's documentation is hosted at [mitiq.readthedocs.io](https://mitiq.readthedocs.io). A PDF version of the latest
 release can be found [here](https://mitiq.readthedocs.io/_/downloads/en/latest/pdf/).
 
+## Examples
+
+Examples on the use of Mitiq are present in the form of Jupyter notebooks in the [examples/](https://github.com/unitaryfund/mitiq/tree/master/examples) repository. They are collected in this [index](examples.html) and rendered as part of the documentation.
```

Comment:
There is no examples.html file. Why not point to RTD here? Examples show up in the table of contents, no?

Reply:
Ah tre: the examples file is generated by nbsphinx, and since the readme is also generated in the docs, this link works from there, but not from github or from the mardown file. Will add a link to the RTD future section.